src/main/java/net/dv8tion/jda/api/sharding/ShardManager.java

```diff
@@ -1160,9 +1161,9 @@ default void setStatusProvider(@Nullable final IntFunction<OnlineStatus> statusP
     /**
      * Initializes and starts all shards. This should only be called once.
      *
-     * @throws LoginException
+     * @throws InvalidTokenException
      *         If the provided token is invalid.
      */
-    void login() throws LoginException;
+    void login() ;
```

Comment:
nit: Remove the space between the parenthesis and the semicolon.

Reply:
So I need to remove all of them right?